browser-test/src/applicant/navigation/northstar_application_navigation_common_intake.test.ts

```diff
@@ -97,13 +97,10 @@ test.describe('Applicant navigation flow', {tag: ['@northstar']}, () => {
         /* wantEligiblePrograms= */ [],
       )
 
-      await validateScreenshot(
-        page,
-        'cif-ineligible-signed-in-confirmation-page',
-        /* fullPage= */ true,
-        /* mobileScreenshot= */ true,
-      )
       await validateAccessibility(page)
+      expect(await page.innerHTML('h2')).toContain(
```

Comment:
**suggestion:** We can remove this (here and below) because it actually already gets done in the helper function called before validateAccessibility - https://github.com/civiform/civiform/blob/3feccd851916271e8781bad5b65b4a8245232c10/browser-test/src/support/applicant_questions.ts#L749-L759

Reply:
Done.